packages/framework/src/Model/Product/Search/ProductSearchRepository.php

```diff
@@ -3,19 +3,17 @@
 namespace Shopsys\FrameworkBundle\Model\Product\Search;
 
 use Doctrine\ORM\QueryBuilder;
-use Shopsys\FrameworkBundle\Component\String\DatabaseSearching;
-use Shopsys\FrameworkBundle\Component\String\TsqueryFactory;
 
 class ProductSearchRepository
 {
     /**
-     * @var \Shopsys\FrameworkBundle\Component\String\TsqueryFactory
+     * @var \Shopsys\FrameworkBundle\Model\Product\Search\SearchClient
```

Comment:
I believe here should be used the cached client instead, or am I missing something? `CachedSearchClient`?

Reply:
oh, I see now, it is set in `services.yml`...